iina/Base.lproj/AboutWindowController.xib

```diff
@@ -200,7 +200,7 @@
                                                     </textField>
                                                     <scrollView wantsLayer="YES" borderType="none" autohidesScrollers="YES" horizontalLineScroll="10" horizontalPageScroll="10" verticalLineScroll="10" verticalPageScroll="10" hasHorizontalScroller="NO" usesPredominantAxisScrolling="NO" translatesAutoresizingMaskIntoConstraints="NO" id="iaQ-kz-jCh">
                                                         <rect key="frame" x="4" y="204" width="376" height="140"/>
-                                                        <clipView key="contentView" drawsBackground="NO" copiesOnScroll="NO" id="WXV-ZB-2K2">
```

Comment:
Please reverse any changes you made in `xib` files.

Reply:
I didn't edit the xib files. If you could point me what to modify I would appreciate it.

Reply:
I changed that line of code.

Reply:
reversed all the modified xib files to the previous state